src/js/views/patients/worklist/worklist_views.js

```diff
@@ -222,6 +225,64 @@ const EmptyFindInListView = View.extend({
   `,
 });
 
+const ListCountTemplate = hbs`
+  <strong>
+    {{#if isFlowList}}
+      {{formatMessage (intlGet "patients.worklist.worklistViews.countView.flowsCount") itemCount=count}}
+    {{else}}
+      {{formatMessage (intlGet "patients.worklist.worklistViews.countView.actionsCount") itemCount=count}}
+    {{/if}}
+  </strong>
+`;
+
+const MaximumCountTemplate = hbs`
+  <div>{{formatMessage (intlGet "patients.worklist.worklistViews.countView.maximumListCount") maximumCount=maximumCount listTypeLabel=listTypeLabel}}</div>
+  <div>{{ @intl.patients.worklist.worklistViews.countView.narrowFilters }}</div>
+`;
+
+const MaximumCountNarrowedTemplate = hbs`
+  <div>
+    {{#if isFlowList}}
+      {{formatMessage (intlGet "patients.worklist.worklistViews.countView.maximumFlowsCountNarrowed") itemCount=count maximumCount=maximumCount}}
+    {{else}}
+      {{formatMessage (intlGet "patients.worklist.worklistViews.countView.maximumActionsCountNarrowed") itemCount=count maximumCount=maximumCount}}
+    {{/if}}
+  </div>
+  <div>{{ @intl.patients.worklist.worklistViews.countView.narrowFilters }}</div>
+`;
+
+const CountView = View.extend({
+  getTemplate() {
+    const filteredCollection = this.getOption('filteredCollection');
+
+    if (!this.collection || !filteredCollection.length) return hbs``;
+
+    const hasReachedMaximum = this.collection.length === MAXIMUM_LIST_COUNT;
+    const isFindInListApplied = hasReachedMaximum && filteredCollection.length < this.collection.length;
+
+    if (!hasReachedMaximum) {
+      return ListCountTemplate;
+    }
+
+    if (isFindInListApplied) {
+      return MaximumCountNarrowedTemplate;
+    }
+
+    return MaximumCountTemplate;
+  },
+  templateContext() {
+    const filteredCollection = this.getOption('filteredCollection');
+    const isFlowList = this.getOption('isFlowList');
+
+    return {
+      maximumCount: MAXIMUM_LIST_COUNT,
+      count: filteredCollection.length,
+      isFlowList,
+      listTypeLabel: isFlowList ? 'Flows' : 'Actions',
```

Comment:
can't do this.  needs to be in the i18n.. but the i18n could have a `{listTypeLabel, select, true "Flows" false "Actions"}` or something like that?  I dunno if it works with boolean..